mage_ai/frontend/components/Triggers/Table.tsx

```diff
@@ -211,6 +232,68 @@ function TriggersTable({
     columnFlex.splice(5, 0, null);
   }
 
+  const [showDisableTriggerModal, hideDisableTriggerModal] = useModal(({ 
+    inProgressRunsCount,
+    left,
+    pipelineScheduleId,
+    top,
+    topOffset,
+  }) => (
+    <DisableTriggerModal 
+      inProgressRunsCount={inProgressRunsCount}
+      left={left} 
+      onAllow={(pipelineScheduleId) => {
+        hideDisableTriggerModal();
+        updatePipelineSchedule({
+          id: pipelineScheduleId,
+          status: ScheduleStatusEnum.INACTIVE,
+        });
+      }}
+      onStop={(pipelineScheduleId) => {
+        hideDisableTriggerModal();
+        // Cancel all in progress runs
+        updatePipeline({
+          pipeline: {
+            pipelineScheduleId: pipelineScheduleId,
```

Comment:
@anniexcheng I believe the object key here should be `pipeline_schedule_id` instead of `pipelineScheduleId`.

Reply:
The reason it's still able to cancel the running pipeline runs is because it's hitting the `else` block of the `cancel_pipeline_runs` method in `PipelineResource`, so it cancels ALL running pipeline runs for the pipeline (not just for the runs under the trigger).

Reply:
oh derp, really good catch!